### ISSUE TITLE
AttributeError: 'Connection' object has no attribute 'connect'

### ISSUE THREAD

**Issue by amrdixon** [dimagi/commcare-export#221]
TL;DR - I believe there might be a bug in the current release of this tool in PyPI, potentially related to SQLAlchemy version updated in git commit 0f8825e in PR #218. Is it possible the current version of the commcare-export tool should be released to PyPI?

Longer explanation:

I'm running the CommCare-export tool via pip, as instructed by the readme, inside a Docker container running Python 3.8. (Dockerfile is [here](https://github.com/dimagi/commcare-export-json/blob/main/Dockerfile))

When I do so, I get the following error, which I think (but am not 100% sure) is a SQLAlchemy version issue already fixed in commit 0f8825e:

commcare_export_json_1  | Traceback (most recent call last):
commcare_export_json_1  |   File "/usr/local/bin/commcare-export", line 8, in <module>
commcare_export_json_1  |     sys.exit(entry_point())
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/commcare_export/cli.py", line 479, in entry_point
commcare_export_json_1  |     main(sys.argv[1:])
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/commcare_export/cli.py", line 217, in main
commcare_export_json_1  |     exit(main_with_args(args))
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/commcare_export/cli.py", line 438, in main_with_args
commcare_export_json_1  |     checkpoint_manager = _get_checkpoint_manager(args)
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/commcare_export/cli.py", line 353, in _get_checkpoint_manager
commcare_export_json_1  |     checkpoint_manager.create_checkpoint_table()
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/commcare_export/checkpoint.py", line 205, in create_checkpoint_table
commcare_export_json_1  |     command.upgrade(cfg, revision)
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/alembic/command.py", line 385, in upgrade
commcare_export_json_1  |     script.run_env()
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/alembic/script/base.py", line 582, in run_env
commcare_export_json_1  |     util.load_python_file(self.dir, "env.py")
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/alembic/util/pyfiles.py", line 94, in load_python_file
commcare_export_json_1  |     module = load_module_py(module_id, path)
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/alembic/util/pyfiles.py", line 110, in load_module_py
commcare_export_json_1  |     spec.loader.exec_module(module)  # type: ignore
commcare_export_json_1  |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
commcare_export_json_1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/migrations/env.py", line 29, in <module>
commcare_export_json_1  |     run_migrations_online()
commcare_export_json_1  |   File "/usr/local/lib/python3.8/site-packages/migrations/env.py", line 19, in run_migrations_online
commcare_export_json_1  |     with connectable.connect() as connection:
commcare_export_json_1  | AttributeError: 'Connection' object has no attribute 'connect'

I was able to get commcare-export to work correctly by creating a tarball of the current main repo and using pip to install directly from the tarball. So, for me, the current version of the tool works.

Is it possible that others are encountering the same issue and a new release to PyPI would help? Thanks!

**Comment by amrdixon** [comment]
This issue was resolved in release 1.10.0 to pypi. Closing the issue.
